ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/UsersAndGroups_Test.java

```diff
@@ -739,22 +740,13 @@ public void putGlobalBucketAclForGroupUUID() throws IOException, SignatureExcept
     @Test
     public void deleteBucketAcl() throws IOException, SignatureException {
         PutBucketAclForUserSpectraS3Response putBucketAclForUserSpectraS3Response = null;
-        try {
-            putBucketAclForUserSpectraS3Response = client.putBucketAclForUserSpectraS3(
-                    new PutBucketAclForUserSpectraS3Request(BUCKET_NAME, BucketAclPermission.DELETE,
-                            spectraUUID));
-            final DeleteBucketAclSpectraS3Response deleteBucketAclSpectraS3Response = client
-                    .deleteBucketAclSpectraS3(new DeleteBucketAclSpectraS3Request(
-                            putBucketAclForUserSpectraS3Response.getBucketAclResult().getId().toString()));
-
-            assertThat(deleteBucketAclSpectraS3Response.getStatusCode(), is(204));
-
-        } finally {
-            if (putBucketAclForUserSpectraS3Response != null) {
-                client.deleteBucketAclSpectraS3(new DeleteBucketAclSpectraS3Request(
+        putBucketAclForUserSpectraS3Response = client.putBucketAclForUserSpectraS3(
```

Comment:
There is no need for the null initialization. These two lines can be combined, and the variable putBucketAclForUserSpectraS3Response can then be made final.